src/vs/workbench/contrib/webviewView/browser/webviewViewPane.ts

```diff
@@ -160,7 +161,7 @@ export class WebviewViewPane extends ViewPane {
 
 		this._activated = true;
 
-		const webviewId = `webviewView-${this.id.replace(/[^a-z0-9]/gi, '-')}`.toLowerCase();
+		const webviewId = generateUuid();
```

Comment:
Is this going to make webview debugging harder because the id is now random? Maybe only resort to a uuid when id is too long to be used as-is.

Reply:
@gjsjohnmurray you still can debug webviews views with
```
	"debugWebviews": true,
	"rendererDebugOptions": {
		"urlFilter": "*vscode-samples.calico-colors*",
		"webRoot": "${workspaceFolder}/media"
	},
```